src/neo4j_genai/retrievers/external/weaviate/weaviate.py

```diff
@@ -130,6 +132,7 @@ def search(
                 logger.debug(
                     "No embedder provided, assuming vectorizer is used in Weaviate."
                 )
+        weaviate_filters = kwargs.get("weaviate_filters")
```

Comment:
Is this line needed?

Reply:
c9081d159d46cdaddbf90285271128c4cf7dc91a